azuread/helpers/validate/strings.go

```diff
@@ -18,3 +19,18 @@ func NoEmptyStrings(i interface{}, k string) ([]string, []error) {
 
 	return nil, nil
 }
+
+// StringIsEmailAddress validates that the given string is a valid email address (foo@bar.com)
+func StringIsEmailAddress(i interface{}, k string) ([]string, []error) {
```

Comment:
Could we get a test for this validation?

Reply:
Test added.